src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewFeatures/FormContext.cs

```diff
@@ -49,8 +80,23 @@ public IList<IHtmlContent> EndOfFormContent
             }
         }
 
+        /// <summary>
+        /// Gets or sets an indication the current &lt;form&gt; is associated with a tag helper or will be generated by
```

Comment:
This kind of goes into implementation details rather than saying what it does. This could simply be `Gets or sets an indication whether extra content can be rendered at the end of the content of this [form] tag.` It can then be followed up with an explanation as to why that might be true or false.